code/modules/mob/living/carbon/human/species/necromorph/necromorph_species.dm

```diff
@@ -318,4 +320,12 @@
 /datum/species/necromorph/handle_post_spawn(var/mob/living/carbon/human/H)
 	.=..()
 	//Apply customisation with a null preference, this applies default settings
-	H.apply_customisation(null)
\ No newline at end of file
+	H.apply_customisation(null)
+
+/mob/living/carbon/human/necromorph/New()
+	. = ..()
+	SSnecromorph.all_necros += src
```

Comment:
major vessels already handles this, and being added to lists is handled through species

the necromorph subtypes of human should not have functionality put on them, they're just convenient entrypoints, everything is handled in species datums